static/src/stylesheets/module/content/_article-immersive.scss

```diff
@@ -18,21 +18,74 @@
     }
 }
 
-.immersive-main-media__img {
+.immersive-main-media__logo {
+    // Position above media
+    z-index: 1;
+}
+
+.immersive-main-media__logo__svg {
+    float: right;
+    margin-top: 6px;
+
+    path {
+        fill: #ffffff;
+    }
+}
+
+.immersive-main-media__media {
     width: 100%;
     // Remove inline spacing
     display: block;
 
+    .element-embed {
+        margin-top: 0;
+        margin-bottom: 0;
+        position: absolute;
+        top: 0;
+        right: 0;
+        bottom: 0;
+        left: 0;
+    }
+
+    .fenced {
+        // Override inline style coming from fence
+        height: 100% !important;
+        opacity: 0;
+        transition: .5s opacity ease-out;
+    }
+
+    .fenced-rendered {
+        opacity: 1;
+    }
+
     @supports (object-fit: cover) and (height: 1vh) and (display: flex) {
         object-fit: cover;
         position: absolute;
         top: 0;
         height: 100%;
         width: 100%;
-        z-index: 0;
     }
 }
 
+.immersive-main-media__media__loading {
+    position: absolute;
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    background-color: $neutral-5;
+    color: $neutral-1;
+}
+
+.immersive-main-media__media__loading__animation {
```

Comment:
Great, thanks!

On Wed, 4 May 2016 at 16:57 Gareth Trufitt <notifications@github.com> wrote:

> In static/src/stylesheets/module/content/_article-immersive.scss
> <https://github.com/guardian/frontend/pull/12775#discussion_r62065527>:
>
> >      }
> >  }
> >
> > +.immersive-main-media__media__loading {
> > +    position: absolute;
> > +    top: 0;
> > +    left: 0;
> > +    right: 0;
> > +    bottom: 0;
> > +    background-color: $neutral-5;
> > +    color: $neutral-1;
> > +}
> > +
> > +.immersive-main-media__media__loading__animation {
>
> I prefer to flatten it, I don't think you gain anything from being
> explicit like that? BEM suggests flattening too:
> http://getbem.com/faq/#css-nested-elements
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/guardian/frontend/pull/12775/files/cc45e8f5bd1ace9d8dd373e02492bb71f1f39d4a#r62065527>
>

-- 

------------------------------
This e-mail and all attachments are confidential and may also be 
privileged. If you are not the named recipient, please notify the sender 
and delete the e-mail and all attachments immediately. Do not disclose the 
contents to another person. You may not use the information for any 
purpose, or store, or copy, it in any way.  Guardian News & Media Limited 
is not liable for any computer viruses or other material transmitted with 
or as part of this e-mail. You should employ virus checking software.
 
Guardian News & Media Limited is a member of Guardian Media Group plc. Registered 
Office: PO Box 68164, Kings Place, 90 York Way, London, N1P 2AP.  Registered 
in England Number 908396